packages/template-retail-react-app/.eslintrc.js

```diff
@@ -6,5 +6,5 @@
  */
 
 module.exports = {
-    extends: require.resolve('pwa-kit-cli/configs/eslint/eslint-config')
+    extends: 'pwa-kit'
```

Comment:
```suggestion
    extends: ['pwa-kit']
```
Very minor, but helps new developers understand you can extend more than one config.